lib/samples/set_basemap/set_basemap_sample.dart

```diff
@@ -115,33 +121,25 @@ class _SetBasemapSampleState extends State<SetBasemapSample>
     );
   }
 
+  void _onMapViewReady() {
+    // Set the map view controller's map to the ArcGIS map.
+    _mapViewController.arcGISMap = _arcGISMap; 
+  }
+
   void updateMap(Basemap basemap) {
+    // Update the map view with the selected basemap.
     _arcGISMap.basemap = basemap;
-    _mapViewController.setViewpointAnimated(_sanFranciscoViewpoint);
+    // Set the rotation angle to 0.
     _mapViewController.setViewpointRotation(angleDegrees: 0.0);
```

Comment:
I'm not sure there's a point to setting the rotation back to 0.0. I think that was just part of the old San Francisco tile map stuff. I think we should get rid of the call to setViewpointRotation.